src/new-client/src/App/Trades/TradesHeader/TradesHeader.tsx

```diff
@@ -2,6 +2,8 @@ import styled from "styled-components"
 import { ExcelButton } from "./ExcelButton"
 import { AppliedFilters } from "./AppliedFilters"
 import { QuickFilter } from "./QuickFilter"
+import { supportsTearOut } from "@/App/supportsTearOut"
+import { TearOutComponent } from "@/Web/TearOutSection/TearOutComponent"
```

Comment:
Can we move all of the `TearOutSection` stuff from web into `App/TearOutSection` please, that way no imports to web specific stuff is happening from `@/App` and will allow us to extend if our other platforms will support tear out (finsemble, glue42 etc..)

Reply:
Moved